src/core/syscalls/syscall_handler.rs

```diff
@@ -2,7 +2,9 @@ use std::any::Any;
 use std::collections::HashMap;
 
 use super::syscall_request::*;
-use super::syscall_response::{GetBlockTimestampResponse, WriteSyscallResponse};
+use super::syscall_response::{
+    GetBlockTimestampResponse, GetSequencerAddressResponse, WriteSyscallResponse,
```

Comment:
in this cases is preferable to use `use super::syscall_response::*` instead of importing to much things.